config/locales/en.yml

```diff
@@ -1603,12 +1603,12 @@ en:
       footer_html: "You can also read the message at %{readurl} and you can send a message to the author at %{replyurl}"
     friendship_notification:
       hi: "Hi %{to_user},"
-      subject: "[OpenStreetMap] %{user} added you as a friend"
-      had_added_you: "%{user} has added you as a friend on OpenStreetMap."
+      subject: "[OpenStreetMap] %{user} followed you"
+      had_followed_you: "%{user} has followed you on OpenStreetMap."
```

Comment:
I'm not sure why this key has a `had_` prefix but it was always incorrect so I'd suggest just dropping it and using `followed_you`.

Reply:
I would probably expect a slightly different wording here, like "%{user} is following you on OpenStreetMap now.". 

"has followed you" sounds to me like this is a thing of the past, and not necessarily the case anymore.

Reply:
`had` was removed and now key is `followed_you`. Also, wording was changed according to suggestions.